### PR TITLE
Breaking sources when wildcard or guava imports present

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -453,11 +453,16 @@
               </includes>
               <trimTrailingWhitespace />
               <endWithNewline />
+              <replaceRegex>
+                <name>Remove wildcard imports</name>
+                <searchRegex>import (static )?\s+\S+\.\*;</searchRegex>
```
Comment (brendandburns) [inline pom.xml:458]: change this to `import (static)?\s+\S+\.\*;(\n);` to pick up the newline

File: pom.xml [hunk 2]
```diff
@@ -453,11 +453,16 @@
...
+                <replacement>INVALID IMPORTS (WILDCARD)</replacement>
```
Comment (brendandburns) [inline pom.xml:459]: Change this to `\n`

**Comment by brendandburns** [comment]
Let's change this so that it just replaces wildcards with a newline. Otherwise LGTM.

File: pom.xml (hunk 2 above)
Comment (brendandburns) [reply pom.xml:459]: I think `$` represents end of expression rather than end of line (`\n` or '\r\n`)

Comment (yue9944882) [reply pom.xml:478]: @brendandburns i tested locally and it seems that the plugin matches the source files by lines so `$` is matching EOL. e.g. adding an `import com.google.guava.*;` line in the middle of the file will be captured.

**Pull request merged**